sdk/attestation/Microsoft.Azure.Management.Attestation/tests/Microsoft.Azure.Management.Attestation.Tests.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$(RepoEngPath)/mgmt/AzSdk.test.reference.props" />
+  <PropertyGroup>
+    <PackageId>Attestation.Tests</PackageId>
+    <Version>1.0.0</Version>
+    <Description>Attestation.Tests Class library</Description>
+    <AssemblyName>Attestatio.Tests</AssemblyName>
+  </PropertyGroup>
+
```

Comment:
Is there any particular reason to remove <TreatWarningsAsErrors>true</TreatWarningsAsErrors>

Reply:
add <TreatWarningsAsErrors>true</TreatWarningsAsErrors> back to .csproj